docs/commerce/storefront/configure_storefront.md

```diff
@@ -0,0 +1,109 @@
+---
+description: Configure Storefront, including catalogs used, customer groups and user accounts.
+edition: commerce
+---
+# Configure Storefront
+
+The Storefront is accessible under the `<yourdomain>/product-catalog`.
+
+## Catalog configuration
+
+With the `ibexa/storefront` package, you can configure the product catalog and make it available to your shop users. 
+
+Before you start configuring the Storefront, make sure you have created and configured [catalogs](https://doc.ibexa.co/projects/userguide/en/latest/pim/work_with_catalogs/#create-catalogs) in the Back Office.
+
+The configuration is available under the `ibexa.system.<scope>.storefront.catalog` key.
+It accepts the following values:
+
+1\. All products available for all users:
+
+```yaml
+ibexa:
+    system:
+        site:
+            storefront:
+                catalog: ~
+```
+
+If `null`is provided as the value, the Storefront makes the main product catalog (with all products) visible for all users.
+
+2\. To expose a single catalog with an identifier to all users, provide a string value of the catalog identifier:
+
+```yaml
+ibexa:
+    system:
+        site:
+            storefront:
+                catalog: custom_catalog
+```
+
+3\. Specific catalog for the defined customer group
+
+
+You can expose different catalogs based on a customer group assigned to the current user.
+
+To do it, provide the following configuration:
+
+```yaml
+ibexa:
+    system:
+        site:
+            storefront:
+                catalog:
+                    default: standard
+                    customer_group:
+                        retailer: retailer_catalog
+                        wholesale: wholesaler_catalog
+```
+
+The basic configuration of the Storefront can look as follows:
+
+``` yaml
+[[= include_file('code_samples/front/shop/storefront/config/packages/ibexa.yaml') =]]
+```
+
+## Retrieve catalog assigned to user
+
+
+The `\Ibexa\Contracts\Storefront\Repository\CatalogResolverInterface` interface allows retrieving the product catalog available for a specific user.
+
+```php
+
+namespace Ibexa\Contracts\Storefront\Repository;
+
+use Ibexa\Contracts\Core\Repository\Values\User\User;
+use Ibexa\Contracts\ProductCatalog\Values\CatalogInterface;
+
+interface CatalogResolverInterface
+{
+    public function resolveCatalog(?User $user = null): ?CatalogInterface;
+}
+```
```

Comment:
```suggestion
```php
namespace Ibexa\Contracts\Storefront\Repository;

use Ibexa\Contracts\Core\Repository\Values\User\User;
use Ibexa\Contracts\ProductCatalog\Values\CatalogInterface;

interface CatalogResolverInterface
{
    public function resolveCatalog(?User $user = null): ?CatalogInterface;
}
```
```

This should fix a syntax highlighting on GH